boost-maven/boost-maven-plugin/src/it/test-mpJwt-1.1/pom.xml

```diff
@@ -76,6 +92,11 @@
     <dependency>
        <groupId>boost.boosters</groupId>
        <artifactId>mpJWT</artifactId>
+    </dependency>
+    <dependency>
+       <groupId>javax.activation</groupId>
+       <artifactId>javax.activation-api</artifactId>
+       <version>1.2.0</version>
```

Comment:
I'm curious what we needed this for, not saying we don't.

Reply:
I'll try running without it and see what happens..